data/tribes/initialization/empire/units.lua

```diff
@@ -1227,13 +1278,21 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_stonemasons_house",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Stonemason's House
+            lore = pgettext("empire_building", "‘marmoris insania!’ (Marble Madness!)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Stonemason's House
+            lore_author = pgettext("A Barabarian about them imperial marbel clumns"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Stonemason's House
             purpose = pgettext("empire_building", "Carves marble columns out of marble.")
          }
       },
       {
          name = "empire_sawmill",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Sawmill
+            lore = pgettext("empire_building", "‘Veni vidi serram duceret’ (I came, I saw, and pulled the saw)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Sawmill
+            lore_author = pgettext("Sawmiller starting to work"),
```

Comment:
The sawmill's worker is called a carpenter. And I think that Latin should be considered the source language here so only puns that work in Latin are ok.